### PR TITLE
chore(docs): update source repository details

### PR DESCRIPTION
**Pull request by kmova**
Refactor and update the source repository information
to reflect the recent reorg of the code based on
different storage engines.

### CODE REVIEW

File: contribute/design/README.md [hunk 1]
```diff
@@ -31,16 +31,72 @@ Storage Device Management Plane can be viewed as an Inventory Management tool, t
...
+- [openebs/performance-benchmark](https://github.com/openebs/performance-benchmark) contains tools/scripts for running performance benchmarks on Kubernetes volumes.  This is an experimental repo. The work in this repo can move to other repos like e2e-tests. 
+- [openebs/monitor-pv](https://github.com/openebs/monitor-pv) is a prometheus exporter for sending capacity usage statistics using `du` from hostpath volumes. 
+- [openebs/helm-operator](https://github.com/openebs/helm-operator) is wrapper around OpenEBS Helm to allow installation via Operator Hub. 
+
+
+
+## Node Disk Manager (aka Storage Device Management plane)
+
+- [openebs/node-disk-manager](https://github.com/openebs/node-disk-manager) contains Kubernetes native Device Inventory Management functionality. Detailed dependency list can be found in [go.mod](https://github.com/openebs/node-disk-manager/blob/master/go.mod). Along with being dependent on Kubernetes and Operator SDK for managing the kubernetes custom resources, NDM also optionally depends on the following.
```
Comment (muratkars) [inline contribute/design/README.md:61]:
```suggestion
- [openebs/node-disk-manager](https://github.com/openebs/node-disk-manager) contains Kubernetes native Device Inventory Management functionality. A detailed dependency list can be found in [go.mod](https://github.com/openebs/node-disk-manager/blob/master/go.mod). Along with being dependent on Kubernetes and Operator SDK for managing the Kubernetes custom resources, NDM also optionally depends on the following. 
```

File: contribute/design/README.md [hunk 2]
```diff
@@ -31,16 +31,72 @@ Storage Device Management Plane can be viewed as an Inventory Management tool, t
...
+
+
+
+## Node Disk Manager (aka Storage Device Management plane)
+
+- [openebs/node-disk-manager](https://github.com/openebs/node-disk-manager) contains Kubernetes native Device Inventory Management functionality. Detailed dependency list can be found in [go.mod](https://github.com/openebs/node-disk-manager/blob/master/go.mod). Along with being dependent on Kubernetes and Operator SDK for managing the kubernetes custom resources, NDM also optionally depends on the following. 
+  - [openSeaChest](https://github.com/Seagate/openSeaChest) (License: MPL 2.0) for discovering device attributes. OpenEBS maintains forked repositories of openSeaChest to fix/upstream the issues found in this libarary.
+
+## Jiva
+- [openebs/jiva](https://github.com/openebs/jiva) is the one of the data engines supported by OpenEBS which was forked from Rancher Longhorn engine and has diverged from the way jiva volumes are managed within Kubernetes. At the time of the fork, Longhorn was focused towards Docker and OpenEBS was focused on supporting Kubernetes. Jiva engine depends on the following:
```
Comment (muratkars) [inline contribute/design/README.md:65]:
```suggestion
- [openebs/jiva](https://github.com/openebs/jiva) is one of the data engines supported by OpenEBS which was forked from Rancher Longhorn engine and has diverged from the way Jiva volumes are managed within Kubernetes. At the time of the fork, Longhorn was focused towards Docker and OpenEBS was focused on supporting Kubernetes. Jiva engine depends on the following:
```

File: contribute/design/README.md [hunk 3]
```diff
@@ -31,16 +31,72 @@ Storage Device Management Plane can be viewed as an Inventory Management tool, t
...
+  - A fork of longhorn engine is maintained in OpenEBS to upsteam the common changes from Jiva to Longhorn.
```
Comment (muratkars) [inline contribute/design/README.md:66]:
```suggestion
  - A fork of Longhorn engine is maintained in OpenEBS to upstream the common changes from Jiva to Longhorn. 
```

File: contribute/design/README.md [hunk 4]
```diff
@@ -31,16 +31,72 @@ Storage Device Management Plane can be viewed as an Inventory Management tool, t
...
+  - [openSeaChest](https://github.com/Seagate/openSeaChest) (License: MPL 2.0) for discovering device attributes. OpenEBS maintains forked repositories of openSeaChest to fix/upstream the issues found in this libarary.
+
+## Jiva
+- [openebs/jiva](https://github.com/openebs/jiva) is the one of the data engines supported by OpenEBS which was forked from Rancher Longhorn engine and has diverged from the way jiva volumes are managed within Kubernetes. At the time of the fork, Longhorn was focused towards Docker and OpenEBS was focused on supporting Kubernetes. Jiva engine depends on the following:
+  - A fork of longhorn engine is maintained in OpenEBS to upsteam the common changes from Jiva to Longhorn. 
+  - [gostor/gotgt](https://github.com/openebs/gotgt) for providing user space iSCSI Target support implemented in Go. A fork of the project is maintained in OpenEBS to keep the dependencies in sync and upstream the changes. 
+  - [longhorn/sparse-tools](https://github.com/openebs/sparse-tools) fork is also maintained by OpenEBS to manage the differences between jiva way of writing into the sparse files.
```
Comment (muratkars) [inline contribute/design/README.md:68]:
```suggestion
  - [longhorn/sparse-tools](https://github.com/openebs/sparse-tools) fork is also maintained by OpenEBS to manage the differences between Jiva way of writing into the sparse files.
```

File: contribute/design/README.md [hunk 5]
```diff
@@ -31,16 +31,72 @@ Storage Device Management Plane can be viewed as an Inventory Management tool, t
...
+OpenEBS source code is spread across multiple repositories, organized either by the storage engine or management layer. This section describes the various actively maintained repositories. 
+
+## Common or Generic 
+
+- [openebs/openebs](https://github.com/openebs/openebs) is OpenEBS meta repository that contains design documents, project management, community and contributor documents, deployment and workload examples. 
+- [openebs/openebs-docs](https://github.com/openebs/openebs-docs) contains the source code for OpenEBS Documentation portal (https://docs.openebs.io) implemented using Docusaurus framework and other libraries listed in [package.json](https://github.com/openebs/openebs-docs/blob/staging/website/package.json)
+- [openebs/website](https://github.com/openebs/website) contains the source code for OpenEBS portal (https://openebs.io) implemented using Gatsby framework and other libraries listed in [package.json](https://github.com/openebs/website/blob/refactor-to-ghost-and-gatsby/package.json)
+- [openebs/charts](https://github.com/openebs/charts) contains the helm chart source code for OpenEBS and also hosts a gh-pages website for install artifacts and helm packages.
```
Comment (muratkars) [inline contribute/design/README.md:41]:
```suggestion
- [openebs/charts](https://github.com/openebs/charts) contains the Helm chart source code for OpenEBS and also hosts a gh-pages website for installing artifacts and Helm packages. 
```

File: contribute/design/README.md [hunk 6]
```diff
@@ -31,16 +31,72 @@ Storage Device Management Plane can be viewed as an Inventory Management tool, t
...
+- [openebs/e2e-tests](https://github.com/openebs/e2e-tests) contains the Litmus based e2e tests that are executed on GitLab pipelines. Contains tests for Jiva, cStor and Local PV.
+
+## Experimental repos  
+
+- [openebs/openebsctl](https://github.com/openebs/openebsctl) contains the OpenEBS CLI that can be run as kubectl plugin. This functionality is being split from the mono-repo [openebs/maya](https://github.com/openebs/maya) into its own repository. (Currently in Alpha). 
+- [openebs/performance-benchmark](https://github.com/openebs/performance-benchmark) contains tools/scripts for running performance benchmarks on Kubernetes volumes.  This is an experimental repo. The work in this repo can move to other repos like e2e-tests. 
+- [openebs/monitor-pv](https://github.com/openebs/monitor-pv) is a prometheus exporter for sending capacity usage statistics using `du` from hostpath volumes. 
+- [openebs/helm-operator](https://github.com/openebs/helm-operator) is wrapper around OpenEBS Helm to allow installation via Operator Hub.
```
Comment (muratkars) [inline contribute/design/README.md:55]:
```suggestion
- [openebs/helm-operator](https://github.com/openebs/helm-operator) is wrapper around OpenEBS Helm to allow installation via the [Operator Hub](https://operatorhub.io/). 
```

**Review by muratkars — CHANGES_REQUESTED** [review]

**Comment by muratkars** [comment]
Thanks for the updates @kmova - I went through the list, all LGTM if you can include minor suggestions I've made.

File: contribute/design/README.md (hunk 5 above)
Comment (pgburt) [inline contribute/design/README.md:41]:
```suggestion
- [openebs/openebs](https://github.com/openebs/openebs) is OpenEBS meta repository that contains design documents, project management, community and contributor documents, deployment and workload examples. 
- [openebs/openebs-docs](https://github.com/openebs/openebs-docs) contains the source code for OpenEBS Documentation portal (https://docs.openebs.io) implemented using Docusaurus framework and other libraries listed in [package.json](https://github.com/openebs/openebs-docs/blob/staging/website/package.json).
- [openebs/website](https://github.com/openebs/website) contains the source code for OpenEBS portal (https://openebs.io) implemented using Gatsby framework and other libraries listed in [package.json](https://github.com/openebs/website/blob/refactor-to-ghost-and-gatsby/package.json).
- [openebs/charts](https://github.com/openebs/charts) contains the helm chart source code for OpenEBS and also hosts a gh-pages website for install artifacts and helm packages. 
```

Comment (pgburt) [reply contribute/design/README.md:41]: We should be consistent with line endings. Either all of the bullet points have line endings, or none should have line endings.

File: contribute/design/README.md [hunk 7]
```diff
@@ -31,16 +31,72 @@ Storage Device Management Plane can be viewed as an Inventory Management tool, t
...
+
+
+## Dynamic Local PVs
+
+- [openebs/zfs-localpv](https://github.com/openebs/zfs-localpv) contains the CSI driver for provisioning Kubernetes Local Volumes on ZFS installed on the nodes. 
+- [openebs/rawfile-localpv](https://github.com/openebs/rawfile-localpv) contains the CSI driver for provisioning Kubernetes Local Volumes on Hostpath by creating sparse files. (Currently in Alpha). 
+
+
 
 # Further details
```
Comment (pgburt) [inline contribute/design/README.md:101]: I wonder if we should encourage people to submit a question or open an issue if they've read this far. Sort of like, giving them a template similar to this https://github.com/openebs/openebs/issues/new?template=feature-request.md 

The current document educates them, but does not give them a way to get involved in any of the projects. Maybe that's something we'll handle in a future tweak? Seeding the idea here, for now :)

Comment (kmova) [reply contribute/design/README.md]: Good one @pgburt. Added a new section.

**Review by muratkars — APPROVED** [review]
LGTM

**Pull request merged**